Add general `extract_coordinate_points` preprocessor (#1581)
ESMValGroup/ESMValCore: doc/recipe/preprocessor.rst

* area_statistics_: Compute area statistics.

``extract``
------------------

> This function extracts points in any given coordinate, following either a 

Reviewer: ```suggestion
This function extracts points with given coordinates, following either a 
```

``linear`` or a ``nearest`` interpolation scheme.
The resulting point cube will match the respective coordinates to
those of the input coordinates. If the input coordinate is a scalar,
the dimension will be a scalar in the output cube.

Examples:

>     * Extract a point in coordinate `grid_latitude`:

Reviewer: ```suggestion
    * Extract a point from coordinate `grid_latitude`with given coordinate value 26.0:
```
